app/src/main/java/com/kickstarter/services/ApiServiceV2.java

```diff
@@ -73,9 +75,15 @@ Observable<Response<ActivityEnvelope>> activities(@NonNull @Query("categories[]"
   @GET("/v1/users/self")
   Observable<Response<User>> currentUser();
 
+  @GET("/v1/users/self")
+  Observable<Response<User>> currentUser(@Query("oauth_token") String token);
```

Comment:
FYI: This method will be removed on following tickets, the token is gonna be sent as header and not query param.